Sources/SpeziOnboarding/ConsentView/ConsentDocument.swift

```diff
@@ -176,6 +177,28 @@ public struct ConsentDocument: View {
 
 /// Extension of `ConsentDocument` enabling the export of the signed consent page.
 extension ConsentDocument {
+    /// As the `PKDrawing.image()` function automatically converts the ink color dependent on the used color scheme (light or dark mode),
+    /// force the ink used in the `UIImage` of the `PKDrawing` to always be black by adjusting the signature ink according to the color scheme.
+    private var blackInkSignatureImage: UIImage {
+        var updatedDrawing = PKDrawing()
+        
+        for stroke in signature.strokes {
+            let blackStroke = PKStroke(
+                ink: PKInk(stroke.ink.inkType, color: colorScheme == .light ? .black : .white),
```

Comment:
Why do we have a ternary statement here? Wouldn't it make sense to make the color black for all color schemes and remove the color scheme variable? Isn't that the main part of the `blackInkSignatureImage` or am I missing something?

Reply:
I indicated that in the comment above (maybe I wasn't clear enough 😀)
The `PKDrawing.image()` function is "smart" enough to automatically adjust the ink color based on the color scheme, so the ink color is black in case of a light color scheme and the ink color is white in case of a black color scheme.
However, as our exported PDF always has a white background, this is the exact opposite of what we're trying to achieve, we always want a black ink color.
Sadly, I didn't find a way to "force" the `PKDrawing.image()` function to always consider the light mode (e.g. tried via the `.preferredColorScheme` modifier), therefore we have to adjust the ink color to `white` in case of the dark color scheme (which is then inverted to black ink by the `PKDrawing.image()` function)

Reply:
Oh, makes sense. That is a bit counter-intuitive but also smart so dark mode is always supported in all the apps ... thank you for the clarification!